Release 7.2.0 changelog (beta) (#10684)
hashicorp/terraform-provider-google-beta: CHANGELOG.md

## 7.2.0 (Unreleased)

FEATURES:
* **New Data Source:** `google_artifact_registry_python_package` ([#10671](https://github.com/hashicorp/terraform-provider-google-beta/pull/10671))
* **New Data Source:** `google_backup_dr_data_source_references` ([#10672](https://github.com/hashicorp/terraform-provider-google-beta/pull/10672))
* **New Resource:** `google_discovery_engine_acl_config` ([#10680](https://github.com/hashicorp/terraform-provider-google-beta/pull/10680))
* **New Resource:** `google_saas_runtime_unit_kind` ([#10652](https://github.com/hashicorp/terraform-provider-google-beta/pull/10652))

IMPROVEMENTS:
* chronicle: made the `scope_info` field in `google_chronicle_reference_list` configurable ([#10663](https://github.com/hashicorp/terraform-provider-google-beta/pull/10663))
* compute: added `header_action` to `path_matcher` and `default_service` level on `google_compute_region_url_map` resource ([#10665](https://github.com/hashicorp/terraform-provider-google-beta/pull/10665))
* container: added `secret_manager_config.rotation_config` field to `google_container_cluster` resource ([#10659](https://github.com/hashicorp/terraform-provider-google-beta/pull/10659))

> * container: added new fields `memory_manager and `topology_manager` to `node_kubelet_config` block ([#10681](https://github.com/hashicorp/terraform-provider-google-beta/pull/10681))

Reviewer: ```suggestion
* container: added new fields `memory_manager` and `topology_manager` to `google_container_cluster.node_config.kubelet_config` and `google_container_node_pool.node_config.kubelet_config` ([#10681](https://github.com/hashicorp/terraform-provider-google-beta/pull/10681))
```
